Simplyfy

kzmdti opened the pull request.

kzmdti committed:
Aggiunto ActivitySetLocation per impostare le preference dell'utente.
modificato ActivityMaps per impostare la mappa nella posizione preferita dell'utente se il gps è spento

Aggiunte condizione di Creazione della partita solo se sono impostati i valori di Place(latitudine e longitudine sono automatici) e il numero di giocatori. La descrizione è facoltativa

Diversi bug di ActivityMaps fixed

kzmdti committed:
Aggiunto ActivitySetLocation per impostare le preference dell'utente.
modificato ActivityMaps per impostare la mappa nella posizione preferita dell'utente se il gps è spento
Aggiunto la possibilità di accendere il GPS programaticamente.
Separate in classi diverse i metodi comuni dove era possibile o ragionevole farlo.

Aggiunte condizione di Creazione della partita solo se sono impostati i valori di Place(latitudine e longitudine sono automatici) e il numero di giocatori. La descrizione è facoltativa

Diversi bug fixed

kzmdti committed:
Merge branch 'master' into Simplyfy

d-u-d-e commented on app/src/main/java/com/es/findsoccerplayers/position/PositionClient.java:
```diff
@@ -0,0 +1,105 @@
+        }
+    }
+
+    public static void turnGPSon(final Context context){
```
non capisco perché c'è un metodo uguale in GPSUtils. Magari se riesci a rendere anche più chiaro cosa fa questo codice meglio, perché onestamente non capisco senza commenti

kzmdti replied:
GpsUtils era da cancellare ma mi sono dimenticato. Infatto non lo uso mai. Aggiungo i commenti

d-u-d-e commented on app/src/main/java/com/es/findsoccerplayers/ActivityMaps.java:
```diff
@@ -147,16 +146,19 @@ public void onClick(View v) {
+                        PositionClient.turnGPSon(ActivityMaps.this);
+                    } else{
+                        // Go to the new view in the map and change the color
+                        if(!isTracking && !PositionClient.isGpsOFF(getApplicationContext())){
```
la seconda condizione dell'if è vera se no sarebbe entrato nell'if precedente

kzmdti replied:
Giusto!

d-u-d-e commented on app/src/main/java/com/es/findsoccerplayers/ActivityMaps.java:
```diff
@@ -211,24 +217,36 @@ public boolean onOptionsItemSelected(MenuItem item) {
+            }
+        }
+
+        //If I have location access and GPS is on then show a small blue circle to identify my position
```
mmmh, cosa cambia dall'if sopra? Stai ritestando le stesse condizioni

kzmdti replied:
Di quali if parli? quello in riga 222 e quello in riga 229?

d-u-d-e replied:
si

kzmdti committed:
Diversi bug fixed
eleminate classi innutili
 Aggiunti commenti su tutto.

kzmdti committed:
Merge remote-tracking branch 'origin/Simplyfy' into Simplyfy

kzmdti committed:
Diversi bug fixed
eleminate classi innutili
 Aggiunti commenti su tutto.

kzmdti committed:
Diversi bug fixed
eleminate classi innutili
 Aggiunti commenti su tutto.
 Aggiunti elementi a strings.xml
